src/http/request.cr

```diff
@@ -98,6 +98,19 @@ class HTTP::Request
     value
   end
 
+  # Return request host from headers
+  def host
+    host = @headers["Host"]?
+    return unless host
+    index = host.index(":")
+    index ? host[0...index] : host
```

Comment:
I'm not sure whether the `Host` header can ever contain an ipv6 address, (formatted `[1234:5678::1]:80`?) but if it does, this code will break.